documentation/docs/infrastructure/overview.md

```diff
@@ -0,0 +1,125 @@
+# Infrastructure
+
+Besides SAP specific Jenkins library steps and out-of-the-box pipelines, project "Piper" offers also documentation and tooling to start the corresponding Jenkins server with all the configuration required to run project "Piper" pipelines.
+
+The core of the Jenkins infrastructure tooling is a set of [Docker images][docker-images].
+There is a main Docker image containing a preconfigured Jenkins and several tooling images used in the specific project "Piper" steps.
+
+The document and the linked resources explain the various ways of starting such a Jenkins server based on these Docker images.
+
+## Cx Server (Recommended)
```

Comment:
this document repeats parts of guidedtour.md. This is not a bad thing perse, but I was a little confused because I already know parts of the text. Maybe something to improve later?

Reply:
I know but I think it is good to have content explicitly in the guided tour. Parts which I considered irrelevant I moved to here.